dvc/exceptions.py

```diff
@@ -206,6 +207,18 @@ def __init__(self, etag, cached_etag):
         )
 
 
+class FileExistsLocallyError(FileExistsError, DvcException):
+    def __init__(self, path, hint=None):
```

Comment:
Do we really need to make the hint conditional?
What is the case where we don't want the hint?

Reply:
👍  I've added support to `dvc add [target] -o [file] --force` with consistent behavior.
Now the hint is not conditional